src/viz/app.py

```diff
@@ -30,23 +32,19 @@ def get_collections():
 
 
 @capture("action")
-def submit_ingest(n_clicks: int, data_path: str, collection: str):
+def submit_ingest(n_clicks: int, data_path: str, collection: str, extra_params: Dict[str, str]):
     """What happens when you click the Submit button."""
     # Prevent trigger on page load
     if not n_clicks:
         return
     client.ingest(collection, path=data_path, loader="web")
-    # requests.post(...)
-    print(data_path, collection)
+    print(data_path, collection, extra_params)
```

Comment:
@yaronha The key-value pairs are available in this function in `extra_params: Dict[str, str]` note values are always strings even if a number is entered).